rllib/core/rl_module/rl_module.py

```diff
@@ -191,15 +203,22 @@ class RLModuleConfig:
             one-hot encoded observation space of the RLModule because of preprocessing.
         action_space: The action space of the RLModule.
         inference_only: Whether the RLModule should be configured in its inference-only
-            state, in which any components not needed for pure action computing (such as
-            a value function or a target network) might be missing.
+            state, in which those components not needed for action computing (for
+            example a value function or a target network) might be missing.
+            Note that `inference_only=True` AND `learner_only=True` is not allowed.
+        learner_only: Whether this RLModule should only be built on Learner workers, but
```

Comment:
Isn't this a duplicate of `inference-only`? What is an example for a `learner_only=True` and `inference_only=False` vs. `learner_only=False` and `inference-only=True`?

And what is an example for `inference_only=False` and `learner_only=False`?

Reply:
This assumes there is a a further stage next to training (learner) and sampling (envrunner). WHich stage is this?